implementations/rust/ockam/ockam_command/src/util/mod.rs

```diff
@@ -188,19 +190,18 @@ impl<'a> Rpc<'a> {
                 ref tcp,
             } => {
                 let port = node_state.setup()?.default_tcp_listener()?.addr.port();
-                let addr = Address::from((TCP, format!("localhost:{port}")));
-                let addr_str = addr.address();
-                match tcp {
+                let addr_str = format!("localhost:{port}");
+                let addr = match tcp {
                     None => {
                         let tcp = TcpTransport::create(ctx).await?;
-                        tcp.connect(addr_str).await?;
+                        tcp.connect(addr_str).await?
                     }
                     Some(tcp) => {
-                        // Ignore "already connected" error.
-                        let _ = tcp.connect(addr_str).await;
+                        // Create a new connection anyway
+                        tcp.connect(addr_str).await?
```

Comment:
So this won't fail now when trying to connect more than once to the same address?